packages/newspringchurchapp/src/prayer/PrayerTab/PrayerTabView.js

```diff
@@ -0,0 +1,218 @@
+      return (
+        <Query query={GET_SAVED_PRAYERS} fetchPolicy="cache-and-network">
+          {({ data }) => {
+            const prayers = get(data, 'savedPrayers', []);
+            return (
+              <PrayerTab
+                prayers={prayers}
+                query={GET_SAVED_PRAYERS}
+                description={'Pray for your saved prayers'}
+                list={'SavedPrayerList'}
+                title={'My Saved Prayers'}
+                type={'saved'}
+              />
+            );
+          }}
+        </Query>
+      );
+    case 'my-church':
+      return (
+        <Query query={GET_PRAYERS} fetchPolicy="cache-and-network">
+          {({ data }) => {
+            const prayers = get(data, 'prayers', []);
+            return (
+              <PrayerTab
+                prayers={prayers}
+                query={GET_PRAYERS}
+                description={'Pray for the people in our church'}
+                list={'ChurchPrayerList'}
+                title={'My Church'}
+                type={'church'}
+              />
+            );
+          }}
+        </Query>
+      );
+    case 'my-campus':
+      return (
+        <Query query={getUserProfile}>
+          {({
+            data: {
+              currentUser: { profile: { campus: { id } = {} } = {} } = {},
+            } = {},
+            loading,
+          }) => {
+            if (loading) return null;
+            return (
+              <Query
+                query={GET_CAMPUS_PRAYERS}
+                variables={{ campusId: id }}
+                fetchPolicy="cache-and-network"
+              >
+                {({ data }) => {
+                  const prayers = get(data, 'campusPrayers', []);
+                  return (
+                    <PrayerTab
+                      query={GET_CAMPUS_PRAYERS}
+                      prayers={prayers}
+                      description={'Pray for the people at your campus'}
+                      list={'CampusPrayerList'}
+                      title={'My Campus'}
+                      type={'campus'}
+                    />
+                  );
+                }}
+              </Query>
+            );
+          }}
+        </Query>
+      );
+    case 'my-community':
+      return (
+        <Query query={GET_GROUP_PRAYERS} fetchPolicy="cache-and-network">
+          {({ data }) => {
+            const prayers = get(data, 'groupsPrayers', []);
+            return (
+              <PrayerTab
+                prayers={prayers}
+                query={GET_GROUP_PRAYERS}
+                description={'Pray for those people in your community'}
+                list={'GroupPrayerList'}
+                title={'My Community'}
+                type={'community'}
+              />
+            );
+          }}
+        </Query>
+      );
+    default:
+      return null;
+  }
+};
+
+const TabWrapper = ({ component }) => <View>{component}</View>;
+
+TabWrapper.propTypes = {
+  component: PropTypes.func,
```

Comment:
```suggestion
  component: PropTypes.shape({}),
```

Getting a warning when the menu loads. This should fix it, but you should probably actually define the real shape.